scripts/configure_petsc.sh

```diff
@@ -74,26 +74,18 @@ function configure_petsc()
     unset IFS
   fi
 
-  # If HDF5 is not found locally, download it via PETSc (except for Darwin)
+  # If HDF5 is not found locally, download it via PETSc
   if [ -z "$HDF5_STR" ]; then
-    if [[ $(uname) == Darwin ]]; then
-      # HDF5 currently doesn't configure properly on macOS due to a gfortran io
-      # buffer bug: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=102992
-      # So, we won't be downloading it by default for now if it isn't found.
-      # Instead, if a user has it installed somewhere, we'll note the variable
-      # they need to set.
-      echo "INFO: HDF5 library not detected; HDF5 related PETSc features will not be enabled."
-      echo "INFO: Set HDF5_DIR to the location of HDF5 and re-run this script, if desired."
-    else
       HDF5_STR="--download-hdf5=1"
+      HDF5_FORTRAN_STR="--download-hdf5-fortran-bindings=0"
```

Comment:
You could combine these two lines if you like. This should work.

`HDF5_STR="--download-hdf5=1 --download-hdf5-fortran-bindings=0"`

Save one variable

Reply:
Actually configure complains with this, thinking that `--download-hdf5-fortran-bindings=0` is part of the definition of `--download-hdf5`. Setting the variable to 
```
HDF5_STR="--download-hdf5=1 \ --download-hdf5-fortran-bindings=0"
```
also didn't work. Is there another formatting trick I could try here?

Reply:
Yeah, then it is fine. Do not want to spend much time on this.